lib/simperium/index.js

```diff
@@ -56,8 +63,10 @@ function BrowserClient({ appID, token, bucketConfig, database, version }) {
     }.bind(this)
   );
 
-  for (let bucket in bucketConfig) {
-    this.buckets[bucket] = this.client.bucket(bucket);
+  for (let bucketName in bucketConfig) {
+    const bucket = this.client.bucket(bucketName);
+    localQueueStore.restoreTo(bucket);
+    this.buckets[bucketName] = bucket;
```

Comment:
Should we only worry about persisting the `tag` and `note` buckets? Probably doesn't hurt anything to do the preferences one too.

Reply:
Yes, I have trouble imagining a bucket where I wouldn't want offline changes to be synced, so I made this the default behavior for all buckets.